installer/scripts/omsadmin.sh

```diff
@@ -352,6 +367,25 @@ set_proxy_setting()
     fi
 }
 
+set_omsagent_proc_limit()
```

Comment:
I have not tested with numbers as small as 1; I will set a minimum limit of 5 to safeguard against unwise input.